tools/topology/topology2/include/pipelines/cavs/passthrough-wov.conf

```diff
@@ -0,0 +1,144 @@
+#
+# CAVS passthrough capture pipeline
+#
+# A simple passthrough pipeline. All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.passthrough-capture.N.attribute_name".
+#
+# Usage: passthrough-capture pipeline object can be instantiated as:
+#
+# Object.Pipeline.passthrough-capture."N" {
+# 	format		"s16le"
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/common/audio_format.conf>
+<include/components/copier.conf>
+<include/components/micsel.conf>
+<include/components/dummywov.conf>
+<include/components/pipeline.conf>
+<include/controls/bytes.conf>
+<common_definitions.conf>
+
+Class.Pipeline."passthrough-wov" {
```

Comment:
please remove all references to passthrough in this pipeline? There's nothing passthrough about it. Isnt this the detect pipeline?